var/spack/packages/boost/package.py

```diff
@@ -58,7 +58,8 @@ def url_for_version(self, version):
         parts = [str(p) for p in Version(version)]
         dots = ".".join(parts)
         underscores = "_".join(parts)
-        return "http://downloads.sourceforge.net/project/boost/boost/%s/boost_%s.tar.bz2" % (
+        #return "http://downloads.sourceforge.net/project/boost/boost/%s/boost_%s.tar.bz2" % (
+        return "http://downloads.sourceforge.net/project/boost/boost/%s.0/boost_%s_0.tar.bz2" % (
```

Comment:
This will cause things to break if the user specifies a three-digit version number -- can you revert this one?  Many packages depend on more specific versions of boost -- just change `CbtfKrell` to depend on `boost@1.59.0` instead of `boost@1.59`.